client/modules/IDE/components/CollectionList/CollectionListRow.jsx

```diff
@@ -79,7 +80,7 @@ class CollectionListRowBase extends React.Component {
 
   handleCollectionDelete = () => {
     this.closeAll();
-    if (window.confirm(`Are you sure you want to delete "${this.props.collection.name}"?`)) {
+    if (window.confirm(this.props.t('CollectionListRow.ConfirmDelete', { key_label: this.props.collection.name }))) {
```

Comment:
Why does `key_label` have an underscore. I think `keyLabel` would fit in with the rest of the code style?

Reply:
Ok, now it is using Common.DeleteConfirmation so is now unified